eZ/Publish/Core/Persistence/Legacy/Content/Location/Gateway/DoctrineDatabase.php

```diff
@@ -631,6 +666,30 @@ function ($pathString) use ($query, $handler) {
         $query->prepare()->execute();
     }
 
+    private function buildHiddenSubtreeQuery(): QueryBuilder
+    {
+        $query = $this->connection->createQueryBuilder();
+        $expr = $query->expr();
+        $query
+            ->select('node_id')
+            ->from('ezcontentobject_tree', 't')
+            ->join('t', 'ezcontentobject', 'c', 't.contentobject_id = c.id')
+            ->where(
+                $expr->or(
+                    $expr->eq(
+                        't.is_hidden',
+                        $query->createPositionalParameter(true, ParameterType::INTEGER)
```

Comment:
Are you sure about parameter type here ? 

```suggestion
                        $query->createPositionalParameter(true, ParameterType::BOOLEAN)
```

Reply:
I don't think so : https://github.com/ezsystems/ezplatform-kernel/blob/v1.3.17/eZ/Bundle/EzPublishCoreBundle/Resources/config/storage/legacy/schema.yaml#L92

Please resolve conversation if you agree

Reply:
> I think my code is correct in this case : https://github.com/ezsystems/ezplatform-kernel/blob/v1.3.17/eZ/Bundle/EzPublishCoreBundle/Resources/config/storage/legacy/schema.yaml#L92
> 
> Please resolve conversation if you agree, or make a comment otherwise

You shouldn't bind true/false to an integer, so in this case it should be the other way around and only because the schema has mistakes:
```php
$query->createPositionalParameter(1, ParameterType::INTEGER)
```